### ISSUE TITLE
nuimo_app: multiple nuimos threading issue

### ISSUE THREAD

**Issue by commento** [getsenic/senic-hub#244]
Happens 50% of the time when nuimo_app is started with multiple nuimos configuration (at the moment can be done only manually)
```
Connecting Nuimo controller  ee:ce:c0:08:3c:aa <Thread(Thread-2, started -1271925648)>
Connecting Nuimo controller  fd:35:69:1c:a3:64 <Thread(Thread-4, started -1290800016)>
started_connecting Nuimo controller  fd:35:69:1c:a3:64 <Thread(Thread-4, started -1290800016)>
started_connecting Nuimo controller  ee:ce:c0:08:3c:aa <Thread(Thread-2, started -1271925648)>
connect_succeeded Nuimo controller  ee:ce:c0:08:3c:aa <Thread(Thread-2, started -1271925648)>
connect_succeeded Nuimo controller  ee:ce:c0:08:3c:aa <Thread(Thread-4, started -1290800016)>
connect_succeeded Nuimo controller  fd:35:69:1c:a3:64 <Thread(Thread-4, started -1290800016)>
```
As visible from this added traces the BLE connection is started for **two** nuimos on **two** different threads but the connect succeeded is received **three** times. 
The controller `ee:ce:c0:08:3c:aa` after this unexpected double response behaves as it is controlled by two apps (two threads)
to be investigated. It takes place only when two nuimos are configured and it is easily recognisable. 
Has been decided to merge the PR #243 code even if this issue is present so that, in the meantime, it is possible to adapt the setup_app for multiple nuimos and test it on the master branch.

**Comment by commento** [comment]
I discovered that the issue is related to BLE connection procedure. When the services_resolved() method is called before connect() the issue takes place:
```
root@senic-hub-beta:~# /usr/bin/nuimo_app -c /var/data/senic-hub/production.ini
2017-08-04 08:05:22,594 INFO  [phue] Attempting to connect to the bridge...
2017-08-04 08:05:22,595 INFO  [phue] Using ip: 192.168.1.96
2017-08-04 08:05:22,596 INFO  [phue] Using username: 3xuMbR9oTL7HeI9tNeBpEqOWXbdcr4j15uAlLqkm
__init__ Nuimo controller  fd:35:69:1c:a3:64 <_MainThread(MainThread, started -1225097216)>
2017-08-04 08:05:23,139 INFO  [phue] Attempting to connect to the bridge...
2017-08-04 08:05:23,140 INFO  [phue] Using ip: 192.168.1.96
2017-08-04 08:05:23,141 INFO  [phue] Using username: 3xuMbR9oTL7HeI9tNeBpEqOWXbdcr4j15uAlLqkm
__init__ Nuimo controller  ee:ce:c0:08:3c:aa <_MainThread(MainThread, started -1225097216)>
nuimo_apps {'fd:35:69:1c:a3:64': <senic_hub.nuimo_app.NuimoApp object at 0xb4e1e630>, 'ee:ce:c0:08:3c:aa': <senic_hub.nuimo_app.NuimoApp object at 0xb4d5cd70>}
Connecting Nuimo controller  fd:35:69:1c:a3:64 <Thread(Thread-2, started -1271925648)>
started_connecting Nuimo controller  fd:35:69:1c:a3:64 <Thread(Thread-2, started -1271925648)>
Connecting Nuimo controller  ee:ce:c0:08:3c:aa <Thread(Thread-4, started -1290800016)>
started_connecting Nuimo controller  ee:ce:c0:08:3c:aa <Thread(Thread-4, started -1290800016)>
Listening to changes of:  /var/data/senic-hub/nuimo_app.cfg
gatt services_resolved
gatt connect
gatt called connect_succeeded
gatt  connect_succeeded
gatt called services_resolved
gatt services_resolved
connect_succeeded Nuimo controller  fd:35:69:1c:a3:64 <Thread(Thread-4, started -1290800016)>
connect_succeeded Nuimo controller  fd:35:69:1c:a3:64 <Thread(Thread-2, started -1271925648)>
gatt connect
gatt called connect_succeeded
gatt  connect_succeeded
gatt called services_resolved
gatt services_resolved
connect_succeeded Nuimo controller  ee:ce:c0:08:3c:aa <Thread(Thread-4, started -1290800016)>
```

**Comment by commento** [comment]
Solved with gatt library version 0.2.6
